plugins/c9.ide.ui/lib_apf.js

```diff
@@ -3726,7 +3726,7 @@ apf.setNodeValue = function(xmlNode, nodeValue) {
     else {
         var oldValue = xmlNode.nodeValue;
         xmlNode.nodeValue = nodeValue === undefined || nodeValue === null ||
-            nodeValue == NaN ? "" : String(nodeValue);
+            isNaN(nodeValue) ? "" : String(nodeValue);
```

Comment:
Although I note that `NaN == null` is `false`; I assume the aim is to preserve the behaviour (rather than the intention) of the original code?